.semaphore/semaphore.yml

```diff
@@ -61,18 +77,6 @@ blocks:
     execution_time_limit:
       minutes: 10
     task:
-      secrets:
-        - name: aws_credentials
-        - name: dockerhub-semaphore-cred
```

Comment:
>And looks like the config there can be a bit simpler:

I'm not sure as we needed to pull the `dockerhub-semaphore-cred` with semaphore secrets from before.

Reply:
I was suggesting a separate config. The tasks in those pipelines just do stuff with S3. They don't pull private images.